src/Components/DrillPage.js

```diff
@@ -114,7 +114,7 @@ export const DrillPage = props => {
         </View>
       </View>
       <View ref={firstDrill} style={styles.animation}>
-        {drill.type === DrillTypes.FRISBEE && <FrisbeeDrillIllustration drill={drill} />}
+        {drill.type === DrillTypes.FRISBEE && <FrisbeeDrillIllustration drill={drill} navigation={navigation} />}
```

Comment:
The branch is not up-to-date with master, it includes a bunch of changes that are not related to this feature

I think you should clean it with the commands I gave you on Slack